server/util/consistent_hash/consistent_hash_test.go

```diff
@@ -103,8 +103,8 @@ func TestEvenLoadDistribution(t *testing.T) {
 		host := ch.Get(string(buf))
 		freq[host]++
 	}
-	minFreq := slices.Min(maps.Values(freq))
-	maxFreq := slices.Max(maps.Values(freq))
+	minFreq := slices.Min(slices.Collect(maps.Values(freq)))
```

Comment:
Does `Min` require collecting into a slice? Not that it matters here, but I would have expected a way to compute the min of a sequence.

Reply:
slices.Min takes a slice not a sequence unfortunately. I was not able to find a golang method to calculate min/max directly from a iterator.